README.md

```diff
@@ -94,6 +97,30 @@ True on cache hit (even if the subsequent
 failed) and false on cache miss. See also
 [skipping steps based on cache-hit](https://github.com/marketplace/actions/cache#Skipping-steps-based-on-cache-hit).
 
+## Anonymous image names
+
+Anonymous images (eg. `<none>:<none>`) will be skipped during caching
+via `docker save`, as we use the `{{ .Repository }}:{{ .Tag }}` format
+to reference which images to save.
+
+`docker save {{ .ID }}` would allow us to cache/load such anonymous images.
+However, the docker client loses reference to the original
+`{{ .Repository }}:{{ .Tag }}` value upon `docker load` for previously
+named images, resulting in:
+
+```sh
+my-image:tag
+```
+
+to become
+
+```sh
+<none>:<none>
+```
+
```

Comment:
This is great context for the commit message, but I doubt the typical developer cares to know our rationale, and I am sensitive to how quickly people stop reading once they feel the docs are no longer relevant.